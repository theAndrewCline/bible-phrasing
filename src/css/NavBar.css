#NavBar {
  height: 50px;
  width: 100vw;
  background: #000a12;
  margin-top: 0;
  display: flex;
  justify-content: flex-start;
  align-items: center;
}

#NavBar > h3 {
  color: #428e92;
  padding: 0.5rem;
}

#NavBar > ul {
  color: #428e92;
  font-family: "PT Serif", "serif";
  font-weight: 700;
  list-style: none;
  display: flex;
  justify-content: space-evenly;
  align-items: center;
  height: 50px;
  margin-left: auto;
  margin-right: 15px;
}

#NavBar > ul > li {
  border-radius: 20px;
  padding: 0.5rem;
}
#NavBar > ul > li:hover {
  color: white;
  background: #428e92;
  box-shadow: black 1px 1px 10px;
  transition: 0.25s;
}
