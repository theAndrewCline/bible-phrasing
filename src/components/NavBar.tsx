import React, { Component } from 'react'
import '../css/NavBar.css'

class NavBar extends Component {
  render () {
    return (
      <div id='NavBar'>
      <h3>Bible Phrasing</h3>
        <ul>
          <li>Present</li>
        </ul>
      </div>
    )
  }
}

export default NavBar
