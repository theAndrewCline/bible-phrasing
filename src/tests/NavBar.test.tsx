import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import NavBar from '../components/NavBar'

describe('<NavBar />', () => {
    it('should match snapshot', () => {
        const wrap = shallow(<NavBar />)
        expect(toJson(wrap)).toMatchSnapshot()
    })
})