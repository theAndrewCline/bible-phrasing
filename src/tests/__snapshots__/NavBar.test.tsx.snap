// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavBar /> should match snapshot 1`] = `
<div
  id="NavBar"
>
  <h3>
    Bible Phrasing
  </h3>
  <ul>
    <li>
      Present
    </li>
  </ul>
</div>
`;
